### PR TITLE
Penalty offense prep

### PR DESCRIPTION
**Pull request by Yudai-8-om**
## Description
The striker will be positioned at .5m behind from the ball as a prep.
Line tactics are commented out until they are ready to use

### Test Case 1
1. Run sim
2. check the striker's position

### CODE REVIEW

**Review by CK-Christopher — COMMENTED** [review]
In basic_seek, it crashes unless stp.role.cost is imported. I am not sure why though. I might just be my VM. In the future, the commented out line tactic needs to match its method header. Other than that, it looks good to me.

**Review by CK-Christopher — APPROVED** [review]

File: rj_gameplay/rj_gameplay/gameplay_node.py [hunk 1]
```diff
@@ -8,30 +8,29 @@
```
Comment (kfu02) [inline rj_gameplay/rj_gameplay/gameplay_node.py]: Can you avoid committing to gameplay_node in this PR? You can reset this file to what's in ros2 with https://stackoverflow.com/questions/2364147/how-to-get-just-one-file-from-another-branch

File: rj_gameplay/rj_gameplay/play/penalty_offense.py [hunk 2]
```diff
@@ -1,75 +1,61 @@
...
-        skill_dict = {}
-        for tactic in self.tactics:
-            skill_dict.update(role_results[tactic])
-        return (skill_dict, skills)
+    ) -> List[RobotIntent]:
+
+        if self._state == State.INIT:
+            self.prioritized_tactics = [
+                prep_move.PrepMove(world_state)
+                # assume line tactic is working
```
Comment (kfu02) [inline rj_gameplay/rj_gameplay/play/penalty_offense.py]: can you put the working line tactic in after #1874 is merged?

File: rj_gameplay/rj_gameplay/play/penalty_offense.py [hunk 3]
```diff
@@ -1,75 +1,61 @@
...
+                # line_tactic.LineTactic(world_state),
+                # line_tactic.LineTactic(world_state),
+                # line_tactic.LineTactic(world_state),
+                # line_tactic.LineTactic(world_state),
+                # line_tactic.LineTactic(world_state),
+            ]
+            self.assign_roles(world_state)
+            self._state = State.PREP
```
Comment (kfu02) [inline rj_gameplay/rj_gameplay/play/penalty_offense.py:40]: you should add a while loop here that checks if the number of requested tactics is actually assigned before switching the state, see this line:

https://github.com/RoboJackets/robocup-software/pull/1855/files#diff-a0f92563cb784149768f1724a5d7e152b2ce3b8042d3477f11ce84f823ddb589R111

**Review by kfu02 — CHANGES_REQUESTED** [review]
some small suggestions, overall behaves well

File: rj_gameplay/rj_gameplay/play/penalty_offense.py (hunk 2 above)
Comment (kfu02) [reply rj_gameplay/rj_gameplay/play/penalty_offense.py]: (this can be done after merging this PR if necessary)

**Review by p-nayak11 — APPROVED** [review]
All of the robots behaved in the proper manner and the striker appropriately approaches the ball in the offensive penalty situation.

**Comment by kfu02** [comment]
was used in #1887 , will be cleaned up by #1895  but I'm merging current state in now to give credit to @Yudai-8-om

**Pull request merged**